screens/ActionsType.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigation, Link } from '@react-navigation/native';
+import {  TouchableOpacity} from 'react-native';
+import server from '../common/server';
+import { API, COLORS } from '../common/constants';
+import { useAppDispatch } from '../stores/hooks';
+import {Text, View,  Box, Radio, Button, Image, ScrollView } from 'native-base';
+import BackIcon from '../assets/images/back-icon.svg'
+import Checkbox from 'expo-checkbox';
+
+
+function ActionsType({route}) {
+  const {actionType} = route.params
+  const [results, setResults] = useState([])
+  const [selectedValues, setSelectedValues] = useState([]);
+  const [isChecked, setChecked] = useState(false);
+  const navigation = useNavigation()
+  const dispatch = useAppDispatch();
+
+
+  const getActionsByType = async (actionType: string) => {
+    if (!actionType) return;
+    const resp = await server.get(`${API.actions}/${actionType}`, {dispatch});
+    setResults(resp.data)
```

Comment:
You're not validating if we got a 200 response or an error. In case the backend generates an error, the app will break completely.